aten/src/ATen/native/ConvolutionMM3d.cpp

```diff
@@ -224,9 +237,7 @@ static void slow_conv3d_update_output_frame(
          output_depth * output_height * output_width});
 
     if (bias.defined()) {
-      for (int64_t i = 0; i < n_output_plane; ++i) {
-        output[i].fill_(bias[i].item());
-      }
+      output.copy_(bias.unsqueeze(-1).unsqueeze(-1).unsqueeze(-1));
```

Comment:
This is nice! This avoids the select op, right?

Reply:
Yes, credit goes to @ngimel